forem/forem
# Admin member index - minor UI/a11y fixes

## What type of PR is this? (check all applicable)


## Description

I did a little sweep of what we have so far in the new member index UI, and noticed a couple of issues:

- Bottom pagination not aligned properly
- User name links were missing the `c-link` class and so didn't match the design
- Accessibility linter was complaining about duplicate IDs on the page, caused by hidden vs visible layouts

## Related Tickets & Documents



- Related Issue https://github.com/forem/forem/issues/16911

## QA Instructions, Screenshots, Recordings

- Enable the `member_index_view` feature flag and visit `/admin/users`
- Check links appear as in [the design](https://www.figma.com/file/Ejc7VELuN00n1SZrkUTBcQ/Member-List-View?node-id=710%3A9081)
- Check bottom pagination looks correctly aligned (you will need 50+ users to trigger this UI 😄 )

### UI accessibility concerns?

Fixes a potential a11y issue

## Added/updated tests?

- [X] No, and this is why: Existing tests continue to cover functionality

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [X] This change does not need to be communicated, and this is why not: Part of a larger issue, still behind a feature flag


## [optional] What gif best describes this PR or how it makes you feel?

![polishing metal](https://media.giphy.com/media/WsUvcV3WOZ10E5IFXw/giphy.gif)
